docs/EN/index.md

```diff
@@ -1,7 +1,8 @@
 :::{admonition} Work-In-Progress
 :class: caution
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/FnwFEFUwXE) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that general users use the beta version unless they are prepared to update quickly and frequently and report any issues found.
+**BETA TESTERS** Because this documentation is still under development, if you notice any discrepancies, errors, or edits that need to be made, please submit an [Issue](https://github.com/nightscout/trio-docs/issues/new/choose) on GitHub.
```

Comment:
If you want `BETA TESTERS` on a new line, add `<br>` to the beginning of line 5

<img width="500" alt="Screenshot 2024-06-19 at 2 31 16 PM" src="https://github.com/nightscout/trio-docs/assets/82073483/b7a3deb8-7259-4b43-a2eb-ecdd85283090">